WorkloadManagementSystem/ConfigTemplate.cfg

```diff
@@ -202,10 +202,14 @@ Agents
   {
     PollingTime = 120
   }
+  ##BEGIN StatesMonitoringAgent
   StatesMonitoringAgent
   {
     PollingTime = 120
```

Comment:
This PollingTime is incorrect, and IIRC it's meaningful here. 900 is the "correct" value I believe.

Reply:
Well. This is depend how often we want to send the report. If we want better resolution we have to send the report every 2 minutes. I think now we send every 15 minutes.

Reply:
15 minutes is 900 seconds, which I would also put as a default here since it seems to work decently enough.